### ISSUE TITLE
lint.xml is unused?

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/focus-android#2420]
I think you need a configuration like:
```groovy
    lintOptions {
        lintConfig file("lint.xml")
    }
```

in app/build.gradle but I don't see one. We had this problem on TV.

**Comment by daivat15** [comment]
I looked into this. What you are saying is correct. The file is unused. I built [this](https://github.com/daivat15/focus-android/tree/removeLintTest) branch and it built successfully.

Should I add the configuration to the gradle file?

**Comment by mcomella** [comment]
@ekager ^

**Comment by bbinto** [comment]
needs to happen with Sentry work

**Comment by ekager** [comment]
Closed by #2443!
